autoload/jsdoc.vim

```diff
@@ -9,11 +9,19 @@ set cpo&vim
 
 let g:jsdoc_templates_path = get(g:, 'jsdoc_templates_path', '')
 let g:jsdoc_formatter = get(g:, 'jsdoc_formatter', 'jsdoc')
-let g:jsdoc_lehre_path = get(
-  \ g:,
-  \ 'jsdoc_lehre_path',
-  \ printf('%s/lib/lehre', expand('<sfile>:p:h:h'))
-  \ )
+if has("win64") || has("win32") || has("win16")
+	let g:jsdoc_lehre_path = get(
+	  \ g:,
+	  \ 'jsdoc_lehre_path',
+	  \ printf('%s/lib/node_modules/.bin/lehre.cmd', expand('<sfile>:p:h:h'))
+	  \ )
+else
+	let g:jsdoc_lehre_path = get(
+	  \ g:,
+	  \ 'jsdoc_lehre_path',
+	  \ printf('%s/lib/lehre', expand('<sfile>:p:h:h'))
+	  \ )
+endif
```

Comment:
@Hwlhwlxyz 
Thank you for fixing.

```suggestion
if has("win64") || has("win32") || has("win16")
  let g:jsdoc_lehre_path = get(
    \ g:,
    \ 'jsdoc_lehre_path',
    \ printf('%s/lib/node_modules/.bin/lehre.cmd', expand('<sfile>:p:h:h'))
    \ )
else
  let g:jsdoc_lehre_path = get(
    \ g:,
    \ 'jsdoc_lehre_path',
    \ printf('%s/lib/lehre', expand('<sfile>:p:h:h'))
    \ )
endif
```

Please fix to 2 space indents.

Reply:
Oh, I didn't notice the indent. Now I commit the suggestion. Thanks.